secondattempt.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```

Comment:
Overall, this code looks awesome! Any reason you put it under secondattempt.html instead of index.html?